include/posts.php

```diff
@@ -0,0 +1,44 @@
+<?php
+function getPostInformation ($blogPostID) {
+    $posts = dbQuery("
+    SELECT *
+    FROM blog_post
+    WHERE blogPostID = :Var1
+    ", array("Var1"=>$blogPostID));
+    return $posts->fetch();
+}
+
+function getCommentInformation ($blogPostID) {
+    $posts = dbQuery("
+    SELECT *
+    FROM blog_comments
+    WHERE blogPostID = :Var2
+    ORDER BY commentID DESC
+    ", array("Var2"=>$blogPostID));
+    return $posts->fetchAll();
+    }
+
+function createComment($body, $name, $email, $blogPostID){
+    $create=
+    dbQuery("
+        INSERT INTO blog_comments (commentBody, commentName, commentEmail, blogPostID)
+        VALUES (:body,:name,:email,:blogPostID)",
+        array(
+            "body"=>$body,
+            "name"=>$name,
+            "email"=>$email,
+            "blogPostID"=>$blogPostID
+        ));
+        return $create;
```

Comment:
I think this function would be better if you returned the commentId you just created. There are a few ways to do this. One is to run another query that looks like this:
```
SELECT LAST_INSERT_ID()```